store/types.go

```diff
@@ -0,0 +1,67 @@
+package store
+
+import (
+	"github.com/FourthState/plasma-mvp-sidechain/plasma"
+	ethcmn "github.com/ethereum/go-ethereum/common"
+	"math/big"
+)
+
+// Wallet holds reference to the total balance, unspent, and spent outputs
+// at a given address
+type Wallet struct {
+	Balance *big.Int          // total amount avaliable to be spent
+	Unspent []plasma.Position // position of unspent transaction outputs
+	Spent   []plasma.Position // position of spent transaction outputs
+}
+
+/* Wrap plasma deposit with spend information */
+type Deposit struct {
+	Deposit   plasma.Deposit
```

Comment:
might be nice to embed these struct by removing the field name. Think that requires the rlp encode/decode work around, so not sure if its worth